index.bs

```diff
@@ -661,15 +664,20 @@ interface MLContext {};
     : <dfn>\[[powerPreference]]</dfn> of type [=power preference=]
     ::
         The {{MLContext}}'s [=power preference=].
-    : <dfn>\[[implementation]]</dfn>
-    ::
-        The underlying implementation provided by the User Agent.
 </dl>
 
 <div class="note">
 When the {{[[contextType]]}} is set to [=default-context|default=] with the {{MLContextOptions}}.{{deviceType}} set to [=device-type-gpu|gpu=], the user agent is responsible for creating an internal GPU device that operates within the context and is capable of ML workload submission on behalf of the calling application. In this setting however, only {{ArrayBufferView}} inputs and outputs are allowed in and out of the graph execution since the application has no way to know what type of internal GPU device is being created on their behalf. In this case, the user agent is responsible for automatic uploads and downloads of the inputs and outputs to and from the GPU memory using this said internal device.
 </div>
 
+### The validate {{MLContext}} algorithm ### {#api-mlcontext-validate}
+To <dfn lt="validate MLContext">validate {{MLContext}}</dfn>, given |context|, run these steps:
+1. If |context|.{{[[contextType]]}} is not "[=webgpu-context|webgpu=]" or "[=default-context|default=], return failure.
```

Comment:
For the validation failure, should it return `false`? For validation success, it returns `true`.

WebGPU spec uses "return false" for validation steps, such as [validating GPUBufferDescriptor](https://www.w3.org/TR/webgpu/#abstract-opdef-validating-gpubufferdescriptor).

Reply:
Yes,  we can return `false`, and then check for "if returns false", instead of "if returns failure".